boards/arm/knot_mesh/knot_mesh.dts

```diff
@@ -90,8 +90,8 @@
 
 &i2c0 {
 	status = "ok";
-	sda-pin = <58>;
-	scl-pin = <59>;
+	sda-pin = <26>;
```

Comment:
In the commit message 1b9bcdb6ff16236f621d58d323d0781411461ae7

You are not adding default pins in this commit. You are changing. Please, rewrite the commit message to be more clear.

Reply:
Done